source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -869,7 +873,9 @@ def worker(dep_mode, arr_mode, instance, request, request_id):
             for idx, j in enumerate(local_resp.journeys):
                 j.internal_id = "{resp}-{j}".format(resp=self.nb_kraken_calls, j=idx)
             if 'ridesharing' in dep_mode or 'ridesharing' in arr_mode:
-                _switch_back_to_ridesharing(local_resp)
+                # dep_mode and arr_mode cannot be ridesharing at the same time
```

Comment:
False, it can if you ask for only ridesharing in last and first section mode.

Reply:
ok

Reply:
OK with @TeXitoi we should manage it separatly, then.